app/src/main/java/com/github/neighbortrader/foodboardapp/clientmodel/Offer.java

```diff
@@ -73,7 +75,7 @@ public static Offer createOfferFromJSON(JSONObject jsonObject) {
             LocalDateTime purchaseDate = LocalDateTime.parse(jsonObject.getString("purchaseDate"), dateTimeFormatter);
             LocalDateTime expireDate = LocalDateTime.parse(jsonObject.getString("expireDate"), dateTimeFormatter);
 
```

Comment:
Warum diese Leezeile zum Beispiel 
Assertion
Assertion
Leerzeile 
Return 

-- einfach straight hintereinander

Reply:
A single blank line may also appear anywhere it improves readability, for example between statements to organize the code into logical subsections. A blank line before the first member or initializer, or after the last member or initializer of the class, is neither encouraged nor discouraged.